src/pybind/mgr/prometheus/module.py

```diff
@@ -1283,6 +1283,8 @@ def get_rbd_stats(self) -> None:
         # '*' can be used to indicate all pools or namespaces
         pools_string = cast(str, self.get_localized_module_option('rbd_stats_pools'))
```

Comment:
I don't find any sensible reason to use the localized module option here... Do we want different set of pools per mgr instance? Nonsensical...

Reply:
Yes, changed to `get_module_option`